graphql/index.tsx

```diff
@@ -5428,6 +5448,7 @@ export type LessonFieldPolicy = {
   docUrl?: FieldPolicy<any> | FieldReadFunction<any>
   githubUrl?: FieldPolicy<any> | FieldReadFunction<any>
   id?: FieldPolicy<any> | FieldReadFunction<any>
+  modules?: FieldPolicy<any> | FieldReadFunction<any>
```

Comment:
Is there a way to avoid using `any` so we can get type errors?

Reply:
We use a tool called `graphql-codegen` to generate types & hooks for our GraphQL queries/mutations. This tool generates the file `graphql/index.tsx` by following the configs in this [file](https://github.com/garageScript/c0d3-app/blob/fe3536f5d7689916d30be0aee93ddf27b79afdd2/codegen.yml).

Therefore, we can't control the types or modify it.